### ISSUE TITLE
Strange tagged folder name being created

### ISSUE THREAD

**Issue by excodex** [instaloader/instaloader#936]
Hello Everyone,

I used instaloader on Debian 10 and now moved to Ubuntu 20.04, when I fast-update (-F) any of my profiles, a new folder appears with this name: `_VX33C~T`, but I also have the `:tagged` folder which was created on a Debian 10 machine. 

If I downloaded all the tagged images and there is no new image to download, ":tagged" folder exists but `_VX33C~T` does not, instaloader creates this strange `_VX33C~T` folder and downloads it again. I would like to know why?

When both folders exist, then I delete the `_VX33C~T` folder and run once more the download with -F and --tagged flags it recognises the `:tagged` folder and does not create `_VX33C~T` again.

When I download a new profile on Ubuntu 20.04 which is not yet exists with --tagged, the strange `_VX33C~T` folder does not appear, just the regular `:tagged` folder.

`:tagged` and `_VX33C~T` has the same contents, but not symlinked.

Thanks in advance!

**Comment by excodex** [comment]
Yeah I figured it out a few weeks ago. But I forgot to post the solution. 

So I switched protocols back then when this error occurred from Samba (CIFS) to NFS.

I wasn't satisfied, so I reverted my decision back to Samba and the same thing happens, but this time it was slightly different.

The colon `:` became some strange UTF-8 garbage character. That was when I realized what really happened...

```
**Tl;dr:** If you switch from Samba (CIFS) to NFS or vice versa, your folder names become obfuscated.
```
